host/xtest/xtest_20000.c

```diff
@@ -129,7 +129,7 @@ static TEEC_Result obj_open(TEEC_Session *sess, void *id, uint32_t id_size,
 	op.params[0].tmpref.size = id_size;
 	op.params[1].value.a = flags;
 	op.params[1].value.b = 0;
-	op.params[2].value.a = TEE_STORAGE_PRIVATE;
+	op.params[2].value.a = TEE_STORAGE_PRIVATE_REE;
```

Comment:
great! 
I wonder how i validated this change !
My apologies.
However, i think this change is mandatory as the tests really needs REE_FS storage from optee.